### PR TITLE
Add 'Spreadshirt' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.datenanfragen.de/#!doc=%7B%22slug%22%3A%22spreadshirt%22%2C%22relevant-countries%22%3A%5B%22all%22%5D%2C%22categories%22%3A%5B%22commerce%22%5D%2C%22name%22%3A%22Spreadshirt%22%2C%22address%22%3A%22sprd.net%20AG%5CnGiesserstr.%2027%5Cn04229%20Leipzig%5CnGermany%22%2C%22phone%22%3A%22%2B49%20341%20594005900%22%2C%22fax%22%3A%22%2B49%20341%20594005499%22%2C%22email%22%3A%22privacy%40spreadshirt.net%22%2C%22web%22%3A%22https%3A%2F%2Fwww.spreadshirt.net%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.spreadshirt.de%2Fdatenschutz-C3928%22%2C%22https%3A%2F%2Fwww.spreadshirt.de%2Fimpressum-C1338%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1184%22%5D%2C%22suggested-transport-medium%22%3A%22email%22%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

File: companies/spreadshirt.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "slug": "spreadshirt",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Spreadshirt",
+    "address": "sprd.net AG\nGiesserstr. 27\n04229 Leipzig\nGermany",
```
Comment (rugk) [inline companies/spreadshirt.json:10]:
```suggestion
    "address": "sprd.net AG\nGießerstraße 27\n04229 Leipzig\nGermany",
```

I'd tend to write the full street name here…

**Review by rugk — APPROVED** [review]
Confirmed/checked sources.

File: companies/spreadshirt.json (hunk 1 above)
Comment (mal-tee) [reply companies/spreadshirt.json]: Yeah, that's better!

**Review by mal-tee — APPROVED** [review]
Thank you!

**Pull request merged**